### PR TITLE
Complete candidate route test suite

### PR DESCRIPTION
**Pull request by p0mvn**
Completes: https://linear.app/osmosis/issue/STABI-140/[sqs-automated-tests-in-ci][candidate-routes]

Add the following integration candidate route tests:
- Between each listed token with liquidity and USDC
- Between combinations of tokens in the following groups:
    1. Top 5 by-liquidity
    2. Top 5 by-volume
    3. Five low liquidity (between 5000 and 10000 USD)
    4. Five low volume (between 5000 and 10000 USD)
- Between tokens with a transmitter route
- Between tokens with an Astroport route

Runs these all in-parallel from 4 workers. There are 457 tests being run.

## Testing
```
source tests/venv/bin/activate
make e2e-run-dev
```

### CODE REVIEW

File: scripts/quote.sh [hunk 1]
```diff
@@ -1,15 +1,17 @@
 #!/bin/bash
 
-url=$1
+sqs_address=$1
+node_address=$2
```
Comment (p0mvn) [inline scripts/quote.sh:4]: Note: this fils is a drive-by change to improve UX around the script

File: tests/setup.py [hunk 2]
```diff
@@ -237,8 +316,64 @@ def chain_denoms_to_display(chain_denoms):
...
+
+    Then,
+    - Puts them all in a set
+    - Constructs combinations between each.
+
+    Returns combinations in the following format:
+    [['denom0', 'denom1']]
+    """
+
+    # Five top by-liquidity tokens
```
Comment (cryptomatictrader) [inline tests/setup.py:353]: Shall we externalize the 5, 5000, 10000 below to constants.py? Or make them as constants in the beginning of the file setup.py, so that they are not like "embedded" in the middle of the code.

File: tests/test_candidate_routes.py [hunk 3]
```diff
@@ -1,39 +1,93 @@
-import requests
 import setup
 import time
+import pytest
 
 from sqs_service import *
 import constants
+from conftest import SERVICE_MAP
 
 SQS_STAGE = "https://sqs.stage.osmosis.zone"
```
Comment (cryptomatictrader) [inline tests/test_candidate_routes.py]: SQS_STAGE and ROUTES_URL also exist in sqs_service.py. Shall we not duplicate them?

**Review by cryptomatictrader — COMMENTED** [review]
Great work! Next we will work on how to integrate it into our stage/prod deployment process. Thank you!

File: tests/test_candidate_routes.py (hunk 3 above)
Comment (p0mvn) [reply tests/test_candidate_routes.py]: Good catch - I think we will need to add a Python linter

Comment (p0mvn) [reply tests/test_candidate_routes.py]: https://github.com/osmosis-labs/sqs/issues/new

File: tests/setup.py (hunk 2 above)
Comment (p0mvn) [reply tests/setup.py:353]: [24b2354](https://github.com/osmosis-labs/sqs/pull/211/commits/24b2354592fc515e6e1977a4339dff665f057fe3)

**Comment by p0mvn** [comment]
Addressed comments, thanks @cryptomatictrader

**Review by cryptomatictrader — COMMENTED** [review]
LGTM 👍

**Review by cryptomatictrader — COMMENTED** [review]
Great work!

**Review by cryptomatictrader — APPROVED** [review]

File: tests/test_candidate_routes.py (hunk 3 above)
Comment (p0mvn) [reply tests/test_candidate_routes.py]: Whoops, I mislinked 😅  - this one: https://github.com/osmosis-labs/sqs/issues/220
